src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -88,17 +87,64 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
           </div>
         </div>
       </div>
-      <div class="width100 pull-left paddingL20 paddingR20">
-          <p class="pull-left margin0 paddingT5" id="wi-detail-assignee-title">
-              Assignee
-          </p>
-          <div class="pull-left detail-assignee-avatar">
-              <span class="pficon-user pull-left" id="wi-detail-assignee"></span>
-          </div>                
+      <div class="width100 pull-left paddingL20 paddingR20 detail-assignee-wrap">
```

Comment:
@sanbornsen I can assign/unassign users without logging in. Please add the loggedIn condition.